op-supervisor/supervisor/backend/cross/safe_update.go

```diff
@@ -118,6 +118,9 @@ func scopedCrossSafeUpdate(h reads.Handle, logger log.Logger, chainID eth.ChainI
 	if err != nil {
 		return candidate, fmt.Errorf("failed to determine dependencies of cross-safe candidate %s: %w", candidate.Derived, err)
 	}
+	if err := SufficientDataChecks(d, candidate.Source.ID(), hazards); err != nil {
+		return candidate, fmt.Errorf("insufficient data to verify block %s: %w", candidate.Derived, err)
+	}
```

Comment:
Do we want to add explicit error handling for an `ErrFuture` case here, or probably better in the caller `CrossSafeUpdate ` above? That's the error we'd get in case there's insufficient data, right? The caller has special handling for other sentinel errors, and the third check is `if !errors.Is(err, types.ErrOutOfScope)` which is what we'd hit I believe. It will probably still lead to overall the right control flow, but we may want to explicitly catch this error and print an `Info` log and return the err, like in the `ErrAwaitReplacementBlock` case.